bindings/wasm/bindings.js

```diff
@@ -596,6 +626,33 @@ Module.setup = function() {
     return out;
   };
 
+  function pushVec3(vec, ps) {
+    toVec(vec, ps, p => {
+      if (p instanceof Array) return {x: p[0], y: p[1], z: p[2]};
+      return p;
+    })
+  }
+
+  Module.Manifold.hull = function(...args) {
+    if (args.length == 1) args = args[0];
+    let pts = new Module.Vector_vec3();
+    for (const m of args) {
+      if (m instanceof ManifoldCtor) {
+        Module._manifoldCollectVertices(pts, m);
+      } else if (
+          m instanceof Array && m.length == 3 && typeof m[0] == 'number') {
+        pts.push_back({x: m[0], y: m[1], z: m[2]});
+      } else if (m.x) {
+        pts.push_back(m);
+      } else {
+        pushVec3(pts, m);
```

Comment:
I feel like we have some helper functions for things like this already - any chance we can make use of those? It also seems a little weird that we can have an array with a manifold and a few extra points, but maybe that'll be useful to someone...

Reply:
They aren't quite the same, polygons2vec is for vectors of vectors.

Reply:
I don't think it's too likely that people would mix manifolds with points, but it was simple enough to just allow the mixing while supporting both lists of points and lists of manifolds separately. 

It might be conventient to hull a single point with a cube to quickly get a roof though for example.